images.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Image struct {
+	Name string
+	Tags []Tag
+}
+
+type Tag struct {
+	Sha string
+	Tag string
+}
+
+var Images = []Image{
+	Image{
+		Name: "quay.io/coreos/hyperkube",
+		Tags: []Tag{
+			Tag{
+				Sha: "5ff22b5c65d5b93aa948b79028dc136a22cda2f049283103f10bd45650b47312",
+				Tag: "v1.5.2_coreos.2",
+			},
+		},
+	},
+}
+
+func RetaggedName(registry, organisation string, image Image) string {
+	parts := strings.Split(image.Name, "/")
+
+	return fmt.Sprintf("%v/%v/%v", registry, organisation, parts[len(parts)-1])
```

Comment:
Hmm, I would argue that godoc for a 2 loc function is not necessary. Also this format is simple. Does it make sense to create a constant for it and have to jump to see it's just `path.Join`?